src/layout/NavBar/components/NavBar.jsx

```diff
@@ -111,10 +111,4 @@ NavBar.propTypes = {
   logout: PropTypes.func.isRequired,
 };
 
```

Comment:
So the linter got mad when we had a requiredProp along with a default value. If the user was not passed, then webpack would not compile altogether, so there was no need to add a default prop value